### ISSUE TITLE
KinematicBody2D (Ball) Stutters and Jumps even when only object in scene.

### ISSUE THREAD

**Issue by AdventurousDrake** [godotengine/godot#17246]
**Godot version: 3.0.1**

**OS/device including version: Windows 10 Home, Nvidia GTX980 (Driver Ver. 391.01)**

**Issue description: KinematicBody2D
I just have a ball bouncing around, no other graphics and the ball stutters and sometimes jumps forward in the direction it is moving. Is this an engine bug? I am using KinematicBody2D, Sprite and CollisionShape2D (Circle). Doesn't seem to matter whether I am using  #_physics_process(delta) or _process(delta). 

I did also code it directly with position += direction * speed * delta, and it was better, but still jumping, but less stuttering. (Compared to move_and_slide which I have now(is worse))

PS. This is my first time to do this, so please contact me if you need more info, Thanks DS**

**Steps to reproduce:
1. Please run code, or executable.

2. See if the ball is stuttering and occasionally jumping in the movement direction, happens quite often, jumping is smaller or bigger but is more arbitrary, should happen within a minute**

**Minimal reproduction project:
[Ball Performance Test Project.zip](https://github.com/godotengine/godot/files/1778448/Ball.Performance.Test.Project.zip)

[test ball performance executable.zip](https://github.com/godotengine/godot/files/1778449/test.ball.performance.executable.zip)**

**Comment by eon-s** [comment]
I cannot see the code or project settings because the examples are compiled, in any case, looks like a duplicate of the old #2043

**Comment by AdventurousDrake** [comment]
Yeah it seems to be the same problem, unfortunately I couldn't find that topic when I searched for KinematicBody2D or I missed it.

I turned off vsync in the project settings, that helped, so it's ok 90% of the time. I've also read that it could maybe be fixed in Godot 3.1. Any release date for it yet?
 
Here is the project, I hope one can read the code now. (If it is still of interest)
[Ball Bouncing Test Project.zip](https://github.com/godotengine/godot/files/1778682/Ball.Bouncing.Test.Project.zip)

Thanks

**Comment by eon-s** [comment]
No ETA for 3.1, and not sure if the work on the physics frame interpolation has started yet (and there are other camera improvements too that may help on this).

Oh, and try adding a background image, sometimes the effects is seen worse over the clear color than with a background texture.

**Comment by Srekel** [comment]
I get jitter with a fairly small example:
`extends KinematicBody2D

var velocity = Vector2(0,0)

func _ready():
	set_physics_process(true)
	
func _physics_process(delta):
	
	var right_input = Input.is_action_pressed("right")
	var left_input = Input.is_action_pressed("left")
	velocity.x = 0
	if right_input:
		velocity.x =150
	if left_input:
		velocity.x = -150

	move_and_slide(velocity, Vector2(0, -1))

`

I recorded a video with OBS and on most frames, my "Player" node moves the same distance, but on a few frames, there is no movement. Presumably this is because sometimes two frames get rendered between two calls to _process I guess. I've set the physics frame rate to 120 but it didn't help. 

[2018-03-16 12-30-24.zip](https://github.com/godotengine/godot/files/1819031/2018-03-16.12-30-24.zip)

So yeah, seems like interpolation is needed. :)
